--- configure
+++ configure
@@ -10488,6 +10488,7 @@
 	       { printf "%s\n" "$as_me:${as_lineno-$LINENO}: result: $ac_res" >&5
 printf "%s\n" "$ac_res" >&6; }
 } # clang
+if false; then
 	{
   ossh_cache_var=ossh_cv_ldflag__Wl__z_retpolineplt
   { printf "%s\n" "$as_me:${as_lineno-$LINENO}: checking if $LD supports link flag -Wl,-z,retpolineplt" >&5
@@ -16154,6 +16155,7 @@
 	       { printf "%s\n" "$as_me:${as_lineno-$LINENO}: result: $ac_res" >&5
 printf "%s\n" "$ac_res" >&6; }
 }
+fi
 	{
   ossh_cache_var=ossh_cv_ldflag__pie
   { printf "%s\n" "$as_me:${as_lineno-$LINENO}: checking if $LD supports link flag -pie" >&5
